scrunch/datasets.py

```diff
@@ -1572,6 +1573,112 @@ def create_categorical(self, categories, alias, name, multiple, description=''):
             return self.create_single_response(
                 categories, alias=alias, name=name, description=description)
 
+    def get_cube(self, dimensions=[], measures=None, filter=[], weight=None):
```

Comment:
Does this function need to be a method of the dataset? 

Could we move this into a `cubes.py` module? And if necessary by the python API then just do `def get_cube(): cubes.get_cube(self)`?

The datasets.py file is growing enormously with dozens of different concerns.